src/Image.cc

```diff
@@ -47,25 +47,38 @@ struct WindowsBitmapFileHeader {
   uint32_t data_offset;
 } __attribute__((packed));
 
+// https://learn.microsoft.com/en-us/windows/win32/api/wingdi/ns-wingdi-bitmapv5header
 struct WindowsBitmapInfoHeader {
-  uint32_t header_size;
-  int32_t width;
-  int32_t height;
-  uint16_t num_planes;
-  uint16_t bit_depth;
-  uint32_t compression; // only BI_RGB (0) and BI_BITFIELDS (3) are supported
-  uint32_t image_size;
-  int32_t x_pixels_per_meter;
-  int32_t y_pixels_per_meter;
-  uint32_t num_used_colors;
-  uint32_t num_important_colors;
-
-  // the following are only present for 32-bit bitmaps apparently
-  uint32_t bitmask_r;
-  uint32_t bitmask_g;
-  uint32_t bitmask_b;
-  uint32_t bitmask_a;
-
+  uint32_t  header_size;
+  int32_t   width;
+  int32_t   height;
+  uint16_t  num_planes;
+  uint16_t  bit_depth;
+  uint32_t  compression; // only BI_RGB (0) and BI_BITFIELDS (3) are supported
+  uint32_t  image_size;
+  int32_t   x_pixels_per_meter;
+  int32_t   y_pixels_per_meter;
+  uint32_t  num_used_colors;
+  uint32_t  num_important_colors;
+
+  // V4 header starts here
+  uint32_t  bitmask_r;
+  uint32_t  bitmask_g;
+  uint32_t  bitmask_b;
+  uint32_t  bitmask_a;
+  uint32_t  colorSpaceType;
+  uint32_t  chromacityEndpoints[9];
+  uint32_t  gammaRed;
+  uint32_t  gammaGreen;
+  uint32_t  gammaBlue;
+  
+  // V5 header starts here
+  uint32_t  renderIntent;
+  uint32_t  colorProfileData;
+  uint32_t  colorProfileSize;
+  uint32_t  reserved;
```

Comment:
Nit: to be consistent with the rest of the structure, these should be snake_case.

Reply:
Sorry for that, I forgot. Fixed.